sherpa/conftest.py

```diff
@@ -407,6 +411,32 @@ def clean_astro_ui():
     # logger.setLevel(old_lgr_level)
 
 
+@pytest.fixture
+def clean_ui():
+    """Ensure sherpa.ui.clean is called before AND after the test.
+
+    See Also
+    --------
+    clean_astro_ui
+
+    Notes
+    -----
+    It does NOT change the logging level; perhaps it should, but the
+    screen output is useful for debugging at this time.
+    """
+    from sherpa import ui
+
+    # old_lgr_level = logger.getEffectiveLevel()
```

Comment:
could we remove these comments from here (and `clean_astro_ui` while we are at it)? If this is useful, e.g. while debugging a test, we could introduce a new fixture for that.

Reply:
I can remove them. Are you okay with the commits being squashed as well?

Reply:
Yes.